ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__ocp-coo-interop.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  observability-operator-test-harness:
+    name: observability-operator-test-harness
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 4G
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: production
+  cron: 0 6 1 7 *
```

Comment:
While the config LGTM, I'm wondering if a *yearly* trigger wouldn't be just too little. Is there any motivation for not triggering this a bit more often?

Reply:
I set it as yearly trigger for I plan to trigger the job with command when an errata is delivered

Reply:
ack, thanks for the heads up! /lgtm then